src/julia.h

```diff
@@ -531,6 +531,7 @@ typedef struct _jl_module_t {
     int8_t compile;
     int8_t infer;
     uint8_t istopmod;
+    uint8_t isprojmod;
```

Comment:
This should probably not be added, since it's not an intrinsic property of a module but can change based on context.